.github/CODEOWNERS

```diff
@@ -594,6 +594,8 @@ embed.go @grafana/grafana-as-code
 /.github/workflows/publish-technical-documentation-release.yml @grafana/docs-grafana
 /.github/workflows/remove-milestone.yml @grafana/user-essentials
 /.github/workflows/sbom-report.yml @grafana/security-team
+/.github/secret_scanning.yml @grafana/security-team
+/.github/workflows/snyk.yml @grafana/security-team
```

Comment:
this is duplicate of https://github.com/grafana/grafana/blob/360c652e39d33c27214a5ccec0176164bc17e740/.github/CODEOWNERS#L603

Reply:
Ah of course. Thanks! I removed the duplicate on line 603